serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/auth/OidcRemoteTokenServices.java

```diff
@@ -56,9 +54,9 @@ public void handleError(ClientHttpResponse response) throws IOException {
             }
         });
 
-        setCheckTokenEndpointUrl(props.getTokenInfoUri());
-        setClientId(props.getClientId());
-        setClientSecret(props.getClientSecret());
+//        setCheckTokenEndpointUrl(props.getTokenInfoUri());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)